src/devices/Media/SoundDevice/WavHeader.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Media
+{
+    internal struct WavHeader
+    {
+        public char[] ChunkId { get; set; }
```

Comment:
Looks like it might be worth to create:

```csharp
struct WavHeaderChunk
{
    public char[] ChunkId { get; set; }
    public uint ChunkSize { get; set; }
}
```

It can be re-used for this chunk and subchunks